src/main/java/com/dre/brewery/configuration/configurer/BreweryXConfigurer.java

```diff
@@ -189,4 +208,69 @@ public String[] getFieldComments(FieldDeclaration fieldDeclaration) {
 			.flatMap(translation -> Arrays.stream(translation.split("\n"))) // Split translations by lines
 			.toArray(String[]::new);
 	}
+
+
+	// Probably could rewrite this null removal to be more efficient...
+
+	public void removeNullValues(Map<String, Object> map) {
+		if (map == null) {
+			return;
+		}
+
+		// Iterate over the map and remove null values
+		Iterator<Map.Entry<String, Object>> iterator = map.entrySet().iterator();
+		while (iterator.hasNext()) {
+			Map.Entry<String, Object> entry = iterator.next();
+			Object value = entry.getValue();
+
+			if (value == null) {
+				iterator.remove(); // Remove entry if value is null
```

Comment:
Removes keys explicitly set to `null`
Not sure if we want that

Reply:
If the null values aren't removed, they're written to the yaml file as the string literal "null".

This (I would assume) causes some read issues for certain fields like 'servercommands'.

Ex:

servercommands: null

^^^^
Okaeri expects a List<String>, not a String


Also, it's kind of unappealing to look at fields being set to "null" rather than them just not being there at all